plugins/repository-gcs/src/main/java/org/elasticsearch/common/blobstore/gcs/GoogleCloudStorageBlobContainer.java

```diff
@@ -85,16 +85,6 @@ public void deleteBlob(String blobName) throws IOException {
     }
 
     @Override
-    public void deleteBlobsByPrefix(String prefix) throws IOException {
-        blobStore.deleteBlobsByPrefix(buildKey(prefix));
-    }
-
-    @Override
-    public void deleteBlobs(Collection<String> blobNames) throws IOException {
-        blobStore.deleteBlobs(buildKeys(blobNames));
```

Comment:
I'm talking about `GoogleCloudStorageBlobStore.deleteBlobs()` method. If it's not used anymore in the `BlobContainer` I think it is useless now.